README.md

```diff
@@ -117,6 +121,33 @@ flutter build ipa
 
 - The Xcode build archive (.xcarchive file) generated can be found inside `[project]/build/ios/archive/` directory and an App Store app bundle (.ipa file) in `[project]/build/ios/ipa/`
 
+## How to Use
+
+### CLN gRPC
+
+- Launch the app
+- In connection type dropdown, select `ClientMode.grpc`
+- Provide the directory path consisting of the TLS Certificate\
+  Example: `/ssd/home/usr/lnd`. The lnd folder consists of the TLS Cert file
```

Comment:
did you use chat gpt to generate this? we do not support `lnd`

Reply:
No I didn't. I guess I did a mistake while checking the directory. It has to be `/home/usr/xyz/.lightning/bitcoin` (incase of mainnet)  and `/home/usr/xyz/.lightning/testnet` (incase of testnet). Am I right?

For my node, it is `/home/usr/xyz/.lightning/testnet`

Reply:
yes you are right